primitives/merkle-mountain-range/src/lib.rs

```diff
@@ -419,6 +423,9 @@ sp_api::decl_runtime_apis! {
 		/// Return the on-chain MMR root hash.
 		fn mmr_root() -> Result<Hash, Error>;
 
+		/// Return the number of MMR blocks in the chain.
+		fn mmr_leaves_count() -> Result<LeafIndex, Error>;
+
```

Comment:
As discussed and tried offline, in case the method is not present in the API we get an `Application(Execution(Other("Exported method MmrApi_mmr_leaves_count is not found")))
`. This is treated by the mmr gadget code, a debug log is printed and the loop that checks for finality notifications continues to run. So we don't need to change the Runtime API version.

Also renamed `mmr_leaves_count` to `mmr_leaf_count`.